### ISSUE TITLE
Group info together

### ISSUE THREAD

**Issue by Donkey-Doug** [Akylas/oss-weather#7]
There is lots of info and some of it is all over the place. There is a section for hourly forcast and a section for daily. These are good organized.

However, the section that shows the current weather is all over the place:

![Screenshot_20221021-200407_OSS Weather](https://user-images.githubusercontent.com/50382414/197260873-ec692bf3-a64e-479d-a19d-815016693955.png)

Some icons/text are on the top, other on the bottom. Some are left aligned, some right.

**Comment by farfromrefug** [comment]
Yes and that it on purpose to create sections. If you have a design please share but remember that it shoould work with any device DPI/size

**Comment by Donkey-Doug** [comment]
To be clear, I like the sections but the screenshot I sent is way too cluttered.

**Comment by RageGamerBoi** [comment]
I think what you want is for the wind speed, moon phase, and precipitation icons to be centered in the blank space, and everything scaled a little bit like this:

![image](https://user-images.githubusercontent.com/105567407/233228198-f9ee191c-9e95-4c72-9433-a23caa45266b.png)

**Comment by farfromrefug** [comment]
@RageGamerBoi i will do something like that. Though so you understand that "blank" space is reserved space for the minutely graph that you get when it is raining (in regions supported by owm or meteo france).

**Comment by RageGamerBoi** [comment]
Oh, I didn't realize that was a feature (which is really cool btw)... Maybe shift the layout back to the current one when the graph needs to display?

**Comment by RageGamerBoi** [comment]
The updated layout looks fantastic!

The one think I think needs to be changed is the temperature needs to be a little bigger for readability purposes.
That's the main metric most people are looking for, and what most weather apps emphasize a little bit more:

![image](https://user-images.githubusercontent.com/105567407/233732231-759ae9e2-f4d5-4629-80e4-26313075b9a8.png)

Idk, just a thought.

**Comment by farfromrefug** [comment]
@RageGamerBoi new build coming. Let me know what you think

**Comment by RageGamerBoi** [comment]
Oh wow, everything is a lot more polished!
It even helped with a little bit of the font scaling problems.
It feels a little empty around the temperature now though, maybe make it even a little bigger and add the unit after just the main readout.
Othar than that, I th

**Comment by farfromrefug** [comment]
@RageGamerBoi thanks i am not finished yet and yes might be a bit empty on the left side. Would prefer not to raise the temp size that big as for me it is far from the most important data. Thinking about those 'icons/data" for wind, uv.... Might want to change those. Also wondering how i could add more (pressure, ...)

**Comment by RageGamerBoi** [comment]
Pressure could be good as an optional item on the graph (like adding an optional traffic layer on top of Google maps), along with wind speed. #31 
Adding the temperature unit could fill the space a bit, and making the current weather icons slightly bigger (when there is no rain graph) could help too.
